java-tools/OsmAndServer/src/main/java/net/osmand/server/index/DownloadIndex.java

```diff
@@ -1,136 +1,130 @@
 package net.osmand.server.index;
 
-import java.io.File;
-import java.io.IOException;
+import net.osmand.server.index.DownloadIndexesService.DownloadType;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
 import java.text.SimpleDateFormat;
 import java.util.Date;
-import java.util.zip.ZipFile;
 
-import javax.xml.stream.XMLStreamException;
-import javax.xml.stream.XMLStreamWriter;
+public class DownloadIndex {
 
-import net.osmand.server.index.DownloadIndexesService.DownloadType;
+	@XmlJavaTypeAdapter(DownloadIndexTypeAdapter.class)
+	@XmlAttribute(name = "type")
+	private DownloadType type;
 
+	private long containerSize;
 
-public class DownloadIndex {
+    private long contentSize;
 
-	private static final double MB =  1 << 20;
-	private DownloadType type;
-	private File file;
-	
-    private long contentSize = -1;
-    private String name;
-    private String description = null;
+    private long timestamp;
+
+	@XmlAttribute(name = "date")
+    private String date;
+
+    @XmlJavaTypeAdapter(DownloadIndexSizeAdapter.class)
+	@XmlAttribute(name = "size")
```

Comment:
Again something with spaces / tabs in the same commit!
+    @XmlJavaTypeAdapter(DownloadIndexSizeAdapter.class)
+	@XmlAttribute(name = "size")